### PR TITLE
Added release script

### PR DESCRIPTION
**Pull request by yanngit**
First version of a release script

### CODE REVIEW

File: hammr-release.sh [hunk 1]
```diff
@@ -0,0 +1,317 @@
...
+
+#Create a new branch because push on master are prohibited ... so we are doing it inside a branch and then merge it inside master
+git checkout -b $HAMMR_RELEASE_BRANCH
+verify_latest_command "Cannot checkout branch"
+
+#Now we change the version of Hammr, SDK and possibly ussclicore
+sed -i "s/'uforge_python_sdk.*'/'uforge_python_sdk==$SDK_VERSION'/g" setup.py
+if [ $CLICORE_VERSION ]; then
+  sed -i "s/'ussclicore.*'/'ussclicore==$CLICORE_VERSION'/g" setup.py
+fi
```
Comment (lqueiroga) [inline hammr-release.sh]: We don't have CLICORE_VERSION finally

Comment (yanngit) [reply hammr-release.sh]: Fixed thanks

**Comment by yanngit** [comment]
Removed ussclicore related stuff

File: hammr-release.sh [hunk 2]
```diff
@@ -0,0 +1,314 @@
+#!/bin/bash
+# What is doing this script ?
+# This script is automating stuff we use to do by hand to release Hammr
+# He's performing in the first place some checks (run inside build-env, not existing directories, existing uforge-python-sdk version ...) to run correctly
```
Comment (thachmai) [inline hammr-release.sh]: Nitpicking here, but a script is an "it" in English not a he :P

**Pull request merged**